crates/subspace-farmer/README.md

```diff
@@ -53,32 +53,18 @@ target/production/subspace-farmer --help
 
 ### Start the farmer
 ```
-target/production/subspace-farmer farm --reward-address st... --plot-size 100G
+target/production/subspace-farmer --farm path=/path/to/disk,size=100G farm --reward-address st...
 ```
 
-`st...` should be replaced with the reward address taken from Polkadot.js wallet (or similar) and `100G` replaced with desired plot size.
+`st...` should be replaced with the reward address taken from Polkadot.js wallet (or similar), `/path/to/disk` with location where you want to store plot and `100G` replaced with desired plot size.
 
 This will connect to local node and will try to solve on every slot notification, while also plotting all existing and new history of the blockchain in parallel.
 
 *NOTE: You need to have a `subspace-node` running before starting farmer, otherwise it will not be able to start*
 
-By default, farmer data are written to `subspace-farmer` subdirectory of the OS-specific users local data directory.
-
-```
-Linux
-$XDG_DATA_HOME or                   /home/alice/.local/share
-$HOME/.local/share 
-
-macOS
-$HOME/Library/Application Support   /Users/Alice/Library/Application Support
-
-Windows
-{FOLDERID_LocalAppData}             C:\Users\Alice\AppData\Local
-```
-
-### Wipe the plot
+### Wipe the plot (same that was created previously)
 ```
-target/production/subspace-farmer wipe
```

Comment:
Do we need to specify the size to wipe it?

Reply:
It wouldn't parse `--farm` otherwise.

Reply:
Can't we allow missing size for `wipe` and forbid it for `farm`? It seems unnatural.

Reply:
Hm... check for `size` is happening on the data structure parsing level, we do not know the command in that context. If we want separate versions of this then we need to move this argument to the command itself (to the right of the command argument), but then it makes little sense to have `--farm path=x`, it should be just `--farm x`, which is inconsistent between commands and I wanted to show that left side from command doesn't change, you just swap `farm`, `info`, `wipe` and it works.

I guess I'm overengineering it :confused:

Reply:
It took me a few reads through but I get the logic now. `--farm <plot config> <command>` is not immediately obvious from `--farm path=/path/to/plot,size=100G farm`. I definitely wasn't expecting the second `farm` on first seeing it though I think that's more about where we're coming from than anything else.

My main worry is that the `wipe` command is useless if I've forgotten the path or plot size I used previously? And does the result of `info` return specific details of the plot at the path I specify? Would it work if I get the size wrong?

If I understand correctly, the intention is that the user can have many plots stored they may want to interact with using `--farm`. It is the farmers responsibility to track their configuration with this implementation.

Reply:
> I definitely wasn't expecting the second farm on first seeing it though I think that's more about where we're coming from than anything else.

Yeah, I agree there is still improvements to be had here.

> My main worry is that the wipe command is useless if I've forgotten the path or plot size I used previously?

That is exactly what we try to avoid by requiring the path explicitly rather than storing files in home directory by default.

> And does the result of info return specific details of the plot at the path I specify?

Of course, what else can it show?

> Would it work if I get the size wrong?

Wipe and info will ignore the size.

> If I understand correctly, the intention is that the user can have many plots stored they may want to interact with using --farm. It is the farmers responsibility to track their configuration with this implementation.

Yes, it somewhat unlikely that they want to store plots in their home directory, they probably have a specific location in mind.